libs/service-portal/health/src/screens/Vaccinations/Vaccinations.graphql

```diff
@@ -0,0 +1,23 @@
+query getVaccinations {
+  getVaccinations {
+    diseaseId
+    diseaseName
+    diseaseDescription
+    vaccinationStatus
+    vaccinationsStatusName
+    lastVaccinationDate
+    vaccinations {
+      id
+      nationalId
+      code
+      vaccinationDate
+      vaccinationsAge {
+        years
+        months
+      }
+      generalComment
+      rejected
+    }
+    comments
+  }
```

Comment:
Query name should also be prefixed with the domain, so `HealthDirectorateVaccinations`. In general, the model being returned should share a name with the query.

Reply:
ok will fix that naming issue after release in another PR 🙏🏼